target/linux/ath79/image/generic.mk

```diff
@@ -308,6 +308,17 @@ define Device/comfast_cf-wr650ac-v2
 endef
 TARGET_DEVICES += comfast_cf-wr650ac-v2
 
+define Device/compex_wpj531-16m
+  ATH_SOC := qca9531
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2
```

Comment:
Remove kmod-usb-core, it is pulled by kmod-usb2.

Reply:
done